<?xml version="1.0"?>

<launch>

  <node name="race_timer_node" pkg="gazebo_tools" type="race_timer_node" output="screen">
    <param name="goal_topic" value="/move_base_simple/goal"/>
    <param name="odom_topic" value="/odom"/>
    <param name="map_frame" value="map"/>
    <param name="use_goal_info" value="false"/>
    <param name="min_time" value="2"/>
    <param name="goal_x" value="5"/>
    <param name="consider_x" value="large"/>
  </node>

</launch>

